examples/plot_sparse_log_reg.py

```diff
@@ -73,14 +74,13 @@
 t0 = time.time()
 
 estimator = LogisticRegression(
-    penalty='l1', fit_intercept=False, solver='saga', max_iter=max_iter)
-model = SparseLogreg(X_train, y_train, max_iter=max_iter, estimator=estimator)
-criterion = HeldOutLogistic(X_val, y_val, model)
+    penalty='l1', solver='saga', fit_intercept=False, max_iter=max_iter)
```

Comment:
celer does not support this solver, I am surprised this work. easy fix: import from sklearn.linear_model